### PR TITLE
Add legacy establishment component

### PR DESCRIPTION
**Pull request by lvnhmd**

### CODE REVIEW

File: client/pages/sections/legacy-establishments.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+import React, {Fragment} from 'react';
+import { connect } from 'react-redux';
+import RepeaterReview from './repeater/review';
+
+const LegacyEstablishments = ({ title, steps, values, readonly, singular, enable,
```
Comment (lennym) [inline client/pages/sections/legacy-establishments.js]: Either line-breaks or no line-breaks, but not this half-way house.

File: client/pages/sections/legacy-establishments.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          <dt>Primary establishment: </dt>
+          <dd>{ name }</dd>
+          <dt>Establishment licence number: </dt>
+          <dd>{ licenceNumber }</dd>
+          <dt>Establishment licence holder: </dt>
+          <dd>{firstName} {lastName}</dd>
+          <dt>Address: </dt>
+          <dd>{ address }</dd>
+        </dl>
+      <RepeaterReview title={title} steps={steps} values={values} readonly={readonly} singular={singular} enable={enable} />
```
Comment (joefitter) [inline client/pages/sections/legacy-establishments.js]: `<RepeaterReview {...props} />`

File: client/pages/sections/legacy-establishments.js (hunk 1 above)
Comment (joefitter) [inline client/pages/sections/legacy-establishments.js]: `({ establishment, ...props })`

File: client/pages/sections/legacy-establishments.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    </Fragment>
+  )
+}
+
+const mapStateToProps = ({ project, application: { readonly, establishment } }) => ({ project, readonly, establishment });
```
Comment (joefitter) [inline client/pages/sections/legacy-establishments.js]: we only need to map the establishment here, not props or readonly as this component doesn't care about them

File: client/pages/sections/legacy-establishments.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+const LegacyEstablishments = ({ title, steps, values, readonly, singular, enable,
+          establishment: {
+            name,
+            licenceNumber,
+            address,
+            licenceHolder: {
+              firstName,
+              lastName
+            }
+          }
```
Comment (joefitter) [inline client/pages/sections/legacy-establishments.js]: I assume this indentation will be fixed :)

File: client/pages/sections/legacy-establishments.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+import React, {Fragment} from 'react';
+import { connect } from 'react-redux';
+import RepeaterReview from './repeater/review';
+
+const LegacyEstablishments = ({ title, steps, values, readonly, singular, enable,
+          establishment: {
+            name,
+            licenceNumber,
+            address,
+            licenceHolder: {
```
Comment (lennym) [inline client/pages/sections/legacy-establishments.js]: You will need to handle the case where this is not set.

File: client/pages/sections/legacy-establishments.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <Fragment>
+      <h2>Primary establishment</h2>
+      <dl className="inline">
+      <dt>Establishment name: </dt>
+      <dd>{ name }</dd>
+      <dt>Licence number: </dt>
+      <dd>{ licenceNumber }</dd>
+      </dl>
+      <br/>
```
Comment (lennym) [inline client/pages/sections/legacy-establishments.js]: This feels like it shouldn't be necessary between block elements. If you want to add whitespace then you should use CSS.

File: client/pages/sections/legacy-establishments.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <Fragment>
+      <div className="granted-section">
+      <h2>Primary establishment</h2>
+      <dl className="inline">
+        <dt>Establishment name: </dt>
+        <dd>{ name }</dd>
+        <dt>Licence number: </dt>
+        <dd>{ licenceNumber }</dd>
+      </dl>
```
Comment (lennym) [inline client/pages/sections/legacy-establishments.js]: The children of the `<div>` should be indented one more level.

**Pull request merged**